### DISCUSSION TITLE
The existence of another ReactPy

### DISCUSSION THREAD

**Discussion by fsmosca** [idom-team/idom#1128] [General]
Just found today, https://github.com/RudreshVeerkhare/ReactPy

**Comment by Archmonger** [comment]
What you know as ReactPy today was previously named IDOM.

At some point we decided ReactPy would be a better name, so we reached out to Rudresh and he was willing to provide us the ReactPy name.

**Reply by fsmosca** [reply]
Thanks for the info.

Looks like Rudresh has not updated its readme then.

**Reply by Archmonger** [reply]
Yep, I don't know if he ever will. It seems like the project is mostly abandoned.

**Comment by DennisHC** [comment]
It seems as though there is another ReactPy repo here as well:
[https://github.com/elilambnz/react-py](https://github.com/elilambnz/react-py)

**Reply by DennisHC** [reply]
After reading through it some more, it seems like the goal is different but the name is similar @Archmonger @fsmosca
